pkg/ddl/ddl_api.go

```diff
@@ -7338,6 +7338,8 @@ func (d *ddl) createIndex(ctx sessionctx.Context, ti ast.Ident, keyType ast.Inde
 		Charset:    chs,
 		Collate:    coll,
 	}
+	job.ReorgMeta.IsDistReorg = variable.EnableDistTask.Load()
+	job.ReorgMeta.IsFastReorg = variable.EnableFastReorg.Load()
```

Comment:
If enbaleDistTask == true and enableFastReorg == false, what will happen?

Reply:
```
mysql> set global tidb_enable_dist_task = 1;
Query OK, 0 rows affected (0.02 sec)

mysql> set global tidb_ddl_enable_fast_reorg = 0;
Query OK, 0 rows affected (0.03 sec)

mysql> alter table t add index i3(c);
Query OK, 0 rows affected (0.43 sec)

mysql> admin show ddl jobs 1;
+--------+---------+------------+---------------------+--------------+-----------+----------+-----------+---------------------+---------------------+---------------------+--------+
| JOB_ID | DB_NAME | TABLE_NAME | JOB_TYPE            | SCHEMA_STATE | SCHEMA_ID | TABLE_ID | ROW_COUNT | CREATE_TIME         | START_TIME          | END_TIME            | STATE  |
+--------+---------+------------+---------------------+--------------+-----------+----------+-----------+---------------------+---------------------+---------------------+--------+
|    111 | test    | t          | add index /* txn */ | public       |         2 |      104 |         3 | 2023-11-16 17:16:52 | 2023-11-16 17:16:52 | 2023-11-16 17:16:53 | synced |
+--------+---------+------------+---------------------+--------------+-----------+----------+-----------+---------------------+---------------------+---------------------+--------+
1 row in set (0.01 sec)
```